src/main/java/uk/gov/digital/ho/hocs/domain/document/JSONToSimpleTextConverter.java

```diff
@@ -58,6 +59,16 @@ private void buildString(JsonNode node, String keyName, int level) {
             convertedOutput.append(String.format("%n%" + (level * 4 - 3) + "s %s%n", "", fromJavaIdentifierToDisplayableString(keyName)));
         } else {
             String textValue = node.textValue();
+            try {
```

Comment:
It would be useful to add a comment to explain which characters are being escaped here